src/nupic/regions/SPRegion.py

```diff
@@ -189,22 +185,19 @@ def getConstraints(arg):
       from highest to lowest. """,
       accessMode='Read',
       dataType='Real32',
-      count=0,
-      constraints=''),
+      count=0 ),
 
     sparseCoincidenceMatrix=dict(
       description='The coincidences, as a SparseMatrix',
       accessMode='Read',
       dataType='Byte',
-      count=0,
-      constraints=''),
+      count=0 ,
```

Comment:
Something is wrong here - the trailing `)` is missing. I don't think that this PR would pass pylint.